### PR TITLE
Refactor openshift certificate roles

### PR DESCRIPTION
**Pull request by abutcher**
And remove openshift certificate logic from playbooks.

### CODE REVIEW

**Comment by abutcher** [comment]
aos-ci-test

**Comment by abutcher** [comment]
aos-ci-test

File: roles/openshift_certificates/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+- fail:
+    msg: "openshift_certificates_ca_host must be defined"
+  when: openshift_certificates_ca_host is not defined
+
+- fail:
+    msg: "openshift_certificates_action must be defined"
+  when: openshift_certificates_action is not defined
```
Comment (detiber) [inline roles/openshift_certificates/tasks/main.yml]: I know I'm backtracking a bit here, but do we get enough code re-use out of consolidating master/node certs? I don't think I'm digging the use of openshift_certificates_action to drive the workflow. 

Maybe if the include behavior was triggered by membership in a list instead? `openshift_certificate_uses = ['master', 'node', 'etcd_server', 'etcd_client']`

Comment (abutcher) [inline roles/openshift_certificates/tasks/main.yml]: Splitting this into separate roles as discussed on IRC. :tada:

**Comment by abutcher** [comment]
aos-ci-test

**Comment by abutcher** [comment]
aos-ci-test

File: roles/openshift_certificates/tasks/main.yml (hunk 1 above)
Comment (abutcher) [inline roles/openshift_certificates/tasks/main.yml]: Reorganized changes pushed up!

**Comment by abutcher** [comment]
aos-ci-test

**Comment by abutcher** [comment]
aos-ci-test

**Comment by detiber** [comment]
@abutcher have you tested this against 1.9.4 and 2.x ? I know there were some changes to the facts that are returned from delegated tasks between the two.

**Comment by abutcher** [comment]
@detiber I haven't tested 1.9.4. CI is covering single master but I'll need to confirm multi.

**Comment by abutcher** [comment]
1.9.4 is A-OK

**Comment by detiber** [comment]
Just holding off on this until after the errata release is cut.

**Comment by detiber** [comment]
@abutcher you good with merging this now?

**Comment by abutcher** [comment]
@detiber Yep, good to merge.

**Pull request merged**